Mathlib/Probability/Kernel/Composition.lean

```diff
@@ -566,62 +567,95 @@ section MapComap
 /-! ### map, comap -/
 
 
-variable {γ δ : Type*} {mγ : MeasurableSpace γ} {mδ : MeasurableSpace δ} {f : β → γ} {g : γ → α}
+variable {γ δ : Type*} [MeasurableSpace γ] {mδ : MeasurableSpace δ} {f : β → γ} {g : γ → α}
```

Comment:
FYI this change broke [the GibbsMeasure project](https://github.com/james18lpc/GibbsMeasure)

Reply:
Reverted in #17299